.bazelrc

```diff
@@ -367,27 +345,25 @@ build:remote-windows --remote_download_toplevel
 build:remote-clang --config=remote
 build:remote-clang --config=rbe-toolchain-clang
 
```

Comment:
Ithink adding one additional newline here would alliminate a diffe between ours and Envoy's `.bazelrc`. Bringing us to a state where all unique lines are marked unique.